fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -1086,6 +1070,15 @@ public void replaceProperties(final IdentifierConverter<Resource, FedoraResource
         }
     }
 
+    private void checkInteractionModel(final Triple triple, final Optional<String> resIxn) {
```

Comment:
For readability, may move this method closer to the other method with the same name, line:894.
https://github.com/fcrepo4/fcrepo4/pull/1442/files#diff-7aee50d5030524028026aa94cd75f350R894